static/js/common/upload-addon.js

```diff
@@ -317,12 +317,14 @@
 
             var $newForm = $('.new-addon-file');
             var $isUnlistedCheckbox = $('#id_is_unlisted');
+            var $channelChoice = $('input[name="channel"]');
 
             function isUnlisted() {
               // True if there's a '#id_is_unlisted' checkbox that is checked, or a
               // 'addon-is-listed' data on the new file form that is true.
               return (($isUnlistedCheckbox.length && $isUnlistedCheckbox.is(':checked')) ||
```

Comment:
`isUnlistedCheckbox` does no longer exist (see comment above). There are other references to it, please remove them as well.

Reply:
That javascript file is used for the primary add-on upload too so make sure whatever is being removed/changed isn't referenced in other templates/forms.

Reply:
@eviljeff  checkbox only exists for the above two pages. So I remove it completely.